build/dependencies.props

```diff
@@ -7,6 +7,7 @@
     <DotNetEfPackageVersion>2.2.0-preview1-34755</DotNetEfPackageVersion>
     <InternalAspNetCoreSdkPackageVersion>2.2.0-preview1-17099</InternalAspNetCoreSdkPackageVersion>
     <MicrosoftAspNetCoreAppPackageVersion>2.2.0-preview1-34755</MicrosoftAspNetCoreAppPackageVersion>
+    <MicrosoftAspNetCoreAllPackageVersion>2.2.0-preview1-34755</MicrosoftAspNetCoreAllPackageVersion>
```

Comment:
This package shouldn't appear in any of our templates. Can we remove all references to .All from the build code?

Reply:
We must have the package [for this to pass](https://github.com/aspnet/Templating/blob/ba02ab85c1250aa6e7a2893518df9a349b993c33/test/GenerateTestProps.targets#L8). You're correct that it shouldn't be in the templates, and it's not. We only use it [here](https://github.com/aspnet/Templating/blob/ba02ab85c1250aa6e7a2893518df9a349b993c33/test/TemplateTests.props.in#L7).

Reply:
Feel free to remove those lines. If we're not running tests or using .All, they don't need to be generated in the test.props file.

Reply:
Why wouldn't we be running tests?